apps/charterafrica/payload.config.js

```diff
@@ -29,7 +30,7 @@ const adapter = s3Adapter({
 export default buildConfig({
   serverURL: appURL,
   collections: [Media, Pages],
-  globals: [FocalCountries, Navigation, Settings],
+  globals: [FocalCountries, Helpdesk, Navigation, Settings],
```

Comment:
In my opinion, I think this could be a `block` instead of a `global`

Reply:
And how would you have the exact content on multiple pages @kelvinkipruto? The same component is in at least 2 pages: homepage and community page.

Reply:
That is ok if the content should be the same on multiple pages. I though each page could have its own content. @kilemensi